### PR TITLE
[skip-ci] Changelog updates

### PR DESCRIPTION
**Pull request by confused-Techie**
This PR includes all our recent changelog updates.

Including the ones we plan to merge prior to release that have not yet.

~~WARNING:
This PR will still need any additions that are including from signing PRs as it seems up in the air about which will go through.~~

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -6,8 +6,59 @@
 
 ## [Unreleased]
 
+## 1.102.0
+
+- Fixed a bug where `pulsar` on Windows could never trigger
+- Fixed `github` package shelling out to `git` on macOS
+- Fixed minor bugs found during fixes to tests
+- Improved our testing infastructure to aide in finding and fixing further bugs
+- Updated many dependencies of Pulsar and it's core packages
```
Comment (Daeraxa) [inline CHANGELOG.md:15]:
```suggestion
- Updated many dependencies of Pulsar and its core packages
```

Comment (confused-Techie) [reply CHANGELOG.md:15]: Thanks for catching this!

File: packages/welcome/lib/changelog-view.js [hunk 2]
```diff
@@ -53,40 +53,37 @@ export default class ChangeLogView {
...
-                Fixed a bug where devtools won't open.
+                Fixed minor bugs found during fixes to tests
               </li>
               <li>
-                Fixed a bug where the editor refused to open with the message "GPU process isn't usable. Goodbye"
+                Improved our testing infastructure to aide in finding and fixing further bugs
               </li>
               <li>
-                Fixed logo artifacts on Linux.
+                Updated many dependencies of Pulsar and it's core packages
```
Comment (Daeraxa) [inline packages/welcome/lib/changelog-view.js:68]:
```suggestion
                Updated many dependencies of Pulsar and its core packages
```

**Review by Daeraxa — APPROVED** [review]

**Comment by Daeraxa** [comment]
Looks good to me now!

**Comment by confused-Techie** [comment]
This PR now includes changes from #387

**Pull request merged**